CenterForOpenScience/osf.io
# [ENG-1994] fix early embargo termination

## Purpose

Embargoes never go away if the the users request early termination and contributors don't reply until after deadline.

## Changes

- add termination statement for registration approval script

## QA Notes

  - Does this change require a data migration? If so, what data will we migrate?
No
  - What is the level of risk?
Low
    - Any permissions code touched?
No
    - Is this an additive or subtractive change, other?
additive
  - How can QA verify? (Through UI, API, AdminApp or AdminAdminApp?)
by attempting to replicate the bug



## Documentation

🐞 fix, no docs

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-1994
